README.md

```diff
@@ -36,7 +36,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: spenserblack/actions-wiki@v0.1.1
+      - uses: spenserblack/actions-wiki@v0.2.0
```

Comment:
Might be easiest on us to just say `@<commit-ish>` or `@main`, and add a comment with a URL to our tags.

Reply:
i think it should be spenserblack@v1 or vN like most other actions to fit the existing ecosystem convention stuff

also `@main` is _sometimes_ broken cause its the in-progress dev-state, but `@vN` or `@vN.N.N` are 100% legit

ideally it'd be `@vN` like `@v1` when we do get to v1 (whenever that happens 🙄) so that it is a general good idea to pin to major versions like that. and also that's what most existing actions do

Reply:
One bonus of `@<commit-ish>` or `@vN` (that's a *literal* `N`, not a placeholder), is that it *forces* a new user to look at our releases instead of just copying and pasting, since it's invalid :laughing: